libs/designsystem/src/lib/components/card/card.component.ts

```diff
@@ -15,6 +16,7 @@ import { ResizeObserverEntry } from '../shared/resize-observer/types/resize-obse
   selector: 'kirby-card',
   templateUrl: './card.component.html',
   styleUrls: ['./card.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
Depending on what constitutes a simple component in this case - is this one too complex due to the sizes input property? 
```
  @Input()
  set sizes(value: { [size: string]: number }) {
    if (typeof value === 'string') {
      console.error(
        'Sizes property cannot be a string. Please ensure the size property is bound as an expression:\n[sizes]="{...}"'
      );
    }
    this.sizesSortedByBreakpoint = this.sortSizesByBreakpoint(value);
  }
```

Reply:
Hmm, my initial hunch was that a setter would ensure change detection being run, but revisiting it now, I am not so sure. Consumers could provide a mutable object here and change it without us knowing, I guess..?

Reply:
Reverted in b6d2f0ac2f1a74768a6bd8b0107c7de8da10a5b4